packages/blade/src/components/ProgressBar/_decisions/decisions.md

```diff
@@ -0,0 +1,102 @@
+# ProgressBar <!-- omit in toc -->
+A Progress bar is generally a branded element that indicates progress of process or task.
+
+<img src="./progress-bar-thumbnail.png" width="380" />
+
+- [Design](#design)
+- [API](#api)
+  - [Sample Usage](#sample-usage)
+  - [Accessibility](#accessibility)
+- [Handling pulsating animation](#handling-pulsating-animation)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+
+## Design
+- [Figma - ProgressBar](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16430%3A256423)
+
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| label | `string` | `undefined` | The label to be rendered with the progress bar. |  |
+| value | number | `undefined` | The current progress value for the progress bar. The progress bar would be in an indeterminate state if `progress` is not passed. |  |
+| contrast | `low`, `high` | `low` | The contrast of the progress bar to be rendered. |  |
+| intent | `positive`,`negative`, `notice`, `information`, `neutral` | `undefined` | The intent of the progress bar to be rendered. | |
+| size | `large`, `small` | `small` | The size of the progress bar to be rendered. |  |
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the progress bar. |
+| isIndeterminate | `boolean` | `false` | Control whether the progress bar is indeterminate or not. The progress bar would be in an indeterminate state if `progress` is not passed.  |
+| showPercentage | `boolean` | `true` | Control whether the progress bar should show the current progress percentage or not. Percentage would be automatically hidden when the progress bar is in an indeterminate state.  |
+
+### Sample Usage
+```jsx
+import { ProgressBar } from '@razorpay/components';
+
+<ProgressBar 
+  label='Loading' 
+  value={20} 
+  contrast='low'
+  size='large'
+  accessibilityLabel='Loading Payments'
+/>
+```
+
+### Accessibility
+- We'll accept a prop `accessibilityLabel` which will be passed on as `aria-label` for web & `accessibilityLabel` for native to aide screen readers
+- By default the value of `accessibilityLabel` will be the same as `label` prop with a fallback to `"Loading"` and can be overwritten with something more meaningful by the consumer.
+- We will set `aria-role` as `progressbar`
+- We will pass the value of `value` prop as `aria-valuenow`
```

Comment:
I've added `aria-busy` following these docs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role#best_practices

Don't think we need anything more since `progressbar` is a distinct role in itself

Reply:
Might need additional things for react native like `accessibilityRole`, `accessibilityState`...

Reply:
Added RN specific things too